modules/nf-core/yak/count/tests/main.nf.test

```diff
@@ -0,0 +1,80 @@
+nextflow_process {
+
+    name "Test Process YAK_COUNT"
+    script "../main.nf"
+    process "YAK_COUNT"
+
+    tag "modules"
+    tag "modules_nfcore"
+    tag "yak"
+    tag "yak/count"
+
+    test("sarscov2 - Illumina - se") {
+
+        when {
+            process {
+                """                
+                input[0] = [
+                    [ id:'test', single_end:true ], // meta map
+                    file(params.modules_testdata_base_path + "genomics/sarscov2/illumina/fastq/test_1.fastq.gz", checkIfExists: true)
+                ]
+                """
+            }
+        }
+
+        then {
+            assertAll(
+                { assert process.success },
+                { assert snapshot(process.out).match() }
+            )
+        }
+
+    }
+
+    test("sarscov2 - Illumina - se - stub") {
```

Comment:
If possible move so it's the final test

Reply:
@Joon-Klaps 
Hi Joon,

Thanks for your very constructive comments and reviews. I might have misunderstood, but did you mean to remove the pe test?

Thanks
Yumi

Reply:
no sorry for being unclear, I meant to move the stub test as a final test in the nf.test file. it's more common to have the stub test just as the last test. So it's 1: se, 2:pe, 3:stub.

Reply:
thanks Joon, all done now.